usdt-impl/src/stapsdt.rs

```diff
@@ -223,9 +223,11 @@ fn compile_probe(
             #[allow(named_asm_labels)]
             unsafe {
                 ::std::arch::asm!(
-                    "990:   nop",
+                    ".Lprobe_{label}: nop",
                     #probe_rec,
                     #in_regs
+                    // hack to get a unique label name.
+                    label = label {},
```

Comment:
this is neat ... and seems like a hack. Why doesn't this cause label to already be defined? This is the first I'm learning of this syntax so certainly don't claim to understand it!